packages/api/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use the official lightweight Node.js 12 image.
+# https://hub.docker.com/_/node
+FROM node:12-slim
+
+# Create and change to the app directory.
+WORKDIR /usr/src/app
+
+# Copy application dependency manifests to the container image.
+# A wildcard is used to ensure both package.json AND package-lock.json are copied.
+# Copying this separately prevents re-running npm install on every code change.
+COPY package*.json ./
+
+# Install production dependencies.
+# Use the official lightweight Node.js 12 image.
+# https://hub.docker.com/_/node
+FROM node:12-slim
+
+# Create and change to the app directory.
+WORKDIR /usr/src/appRUN npm install --only=production
+
+
+# Copy local code to the container image.
+COPY . ./
```

Comment:
We shouldn't need two `COPY` commands either, looks almost like two Dockerfiles might have been added on top of each other 😄 

I think we can pretty much almost use [this exact file](https://github.com/knative/docs/blob/ca0a1b32da4df27a043ebdbd0bd032d1723bbadb/docs/serving/samples/hello-world/helloworld-nodejs/Dockerfile) which it looks like you're close to.

Reply:
Oh yes you are right, so the second COPY command should be removed? The other file you attached has a COPY ../ command and performs COPY package*.json ./ separately

Reply:
Having multiple copy commands is fine, but I think that `COPY .`  would actually copy over `package.json`, maybe?

If things stop working after that change I'm wrong 😆